.env.sample

```diff
@@ -69,4 +69,7 @@ IS_AUTH_TOKEN_BEARER=false
 AUTH_METHOD = native #or keycloak_public_key
 KEYCLOAK_PUBLIC_KEY_PATH = path to the pem/secret file
 VALIDATE_ROLE = 'ON' 
-TOP_LEVEL_ENTITY_TYPE = state
\ No newline at end of file
+TOP_LEVEL_ENTITY_TYPE = state
+
+ELEVATE_SURVEY_SERVICE_URL = https://survey-dev.elevate-apis.shikshalokam.org
```

Comment:
@borkarsaish65 we don't have to add in envVariables.js file?

Reply:
I have added